Sources/Testing/Testing.docc/AddingTags.md

```diff
@@ -16,8 +16,8 @@ Use tags to provide semantic information for organization, filtering, and custom
 
 A complex package or project may contain hundreds or thousands of tests and
 suites. Some subset of those tests may share some common facet, such as being
-"critical" or "flaky". The testing library includes a type of trait called
-"tags" that can be added to tests to group and categorize them.
+*critical* or *flaky*. The testing library includes a type of trait called
```

Comment:
For italics, please use underscores instead of asterisks. Some Markdown parsers render single asterisks as bold.